src/main/java/put/io/black/java/rest/ScenarioController.java

```diff
@@ -0,0 +1,132 @@
+package put.io.black.java.rest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.*;
+import put.io.black.java.logic.ScenarioManager;
+import put.io.black.java.logic.TextTransformer;
+
+import java.util.Arrays;
+
+@RestController
+@RequestMapping("/{text}")
+public class ScenarioController {
+
+    private static final Logger logger = LoggerFactory.getLogger(ScenarioController.class);
+
+    @RequestMapping(method = RequestMethod.GET, produces = "application/json")
+    public String get(@PathVariable String text, @RequestParam(value="transforms", defaultValue="upper,escape") String[] transforms) {
+
+        // log the parameters
+        logger.debug(text);
+        logger.debug(Arrays.toString(transforms));
+
+        // do the transformation, you should run your logic here, below just a silly example
```

Comment:
Delete comments from code. All the same and without affecting the understanding of the code.

Reply:
Cat army killed in Intellij